build/linux/installer/conf/fluent-bit-network-flow-logs.conf

```diff
@@ -0,0 +1,30 @@
+[INPUT]
```

Comment:
as we discussed, we should add throttling support and also implement network flow logs extension DCR.

Reply:
extension not planning to do.